lib/tests/testutil.py

```diff
@@ -53,7 +54,7 @@ def setUp(self, override_root=True):
 
         if override_root:
             main_dg = self.new_delta_generator()
-            sidebar_dg = self.new_delta_generator()
+            sidebar_dg = self.new_delta_generator(container=BlockPath.SIDEBAR)
```

Comment:
This change wasn't integral to my PR but seems like the right thing to do to avoid potential issues with using this class in tests in the future.